realm/realm-library/src/androidTest/java/io/realm/rule/TemporaryRealmFolder.java

```diff
@@ -0,0 +1,66 @@
+package io.realm.rule;
+
+import org.junit.rules.TemporaryFolder;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+
+import io.realm.Realm;
+import io.realm.RealmConfiguration;
+
+/**
+ * Rule that creates {@link RealmConfiguration } in a temporary directory and delete Realms created with those
```

Comment:
"those" -> "this" (there is only one RealmConfiguration - otherwise it should begin by "Rule that created RealmConfigurations")